docs/builtins.md

```diff
@@ -1274,7 +1274,7 @@ from the left edge of the canvas and 60 units from the bottom edge.
 
 <img width="300" alt="Cartesian coordinates with x and y axes and point (30, 60)" src="img/coords.png">
 
-[Evy source](https://evy.dev/#content=H4sIAAAAAAAAE0WRwW6EIBCG7/MUE86NAV1w8dqkL7HxYJFtSVE2rmnXNn33DgL2NDMfA///h7fFjWCCDwuyVz+YDwZfblzfkVcnAO9mixwF5zCFz9jyxIhQC7fgN7zsQ48XfUZRydTEOfG8tL8SYSVRn/sIcokc4BrmFX/u7tt2J7wOk/Nbx17cMuBzGO0TTmEO99tgLPtNXrREAat9rMgeLCFBMKPtQGWJM0ioOVJQp3gJqY48hKUma7VG2VJtRK47z0t0qGKeus3LLSrRF56laomqzvq72OFBYK2L1e6fC5JQJVXXRJ4/Z7EjO/yTV+MW4+kC/AGSFdVbwgEAAA==).
+[Evy source].
```

Comment:
This should probably be named something more specific. Not sure if you can reuse these in different sections, but perhaps `[Evy source:coords]` or something (`[Evy source coords]` even).

That said, I think this one could be an exception since there is no other text. I like footnote style since it makes the text easier to read. That doesn't apply in this case so no need for it if you prefer not to change this one.

Reply:
done.